_spinnaker_install_admin_guides/sumologic-dashboard.md

```diff
@@ -0,0 +1,93 @@
+```
+_sourceCategory="dev/sales-demo-cluster/echo" and _collector="Spinnaker Instance"  orca pipeline starting
+| json field=_raw "content.execution.status","content.execution.application","details.type","content.execution.name","content.execution.trigger.type","content.execution.authentication.user" as status,application,logType,pipeline, triggerType, user nodrop
+| where logType = "orca:pipeline:starting"
+| timeslice by 1d
+| count by _timeslice, pipeline
+| transpose row _timeslice column pipeline
+```
+
+## Collect Logs for Spinnaker
+
+### Collection process overview
+Note: This assumes you have already installed Spinnaker and have access to the halyard configuration files  to make changes to Spinnaker.
+
+### Collection Step 1. Retrieve SumoLogic URL for HTTP logs
+Configure an HTTP Collector. And get a copy of the endpoint URL from your SumoLogic console.
+
+### Collection step 2. Modify Spinnaker echo-local.yml to export logs to SumoLogic
+
+Add (and create if necessary) to `.hal/default/echo-local.yml` the following:
+```yaml
+rest:
+  enabled: true
+  endpoints:
+    - wrap: false
+      url: [insert your custom HTTP endpoint] #e.g. https://endpoint1.collection.us1.sumologic.com/...
+```
+### Collection step 3. Run `hal deploy apply`
+Run `hal deploy apply` and wait for the services to restart.  Check SumoLogic console for ingestion of data.
+
+## Install the Spinnaker App and View the Dashboards
+Go to Sumo Logic App Catalog and search for "Spinnaker" by Armory
+
+### Dashboard filters  
+The Spinnaker Pipelines dashboard has a set of filters that you can apply to the entire dashboard as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+NOTE: You can use filters to drill down and examine the data on a granular level by application and pipeline
+![GlobalDashboardFilter](/assets/images/sumologic-dashboard-filter.png)
+Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+![DashboardFilter](/assets/images/sumologic-dashboard-filter2.png)
+
+### Spinnaker Overview Dashboard
+The Spinnaker Overview provides the health and usage of your Spinnaker instance at a quick glance, including applications deployed, clusters targeted, pipelines run, and users in spinnaker.
+Use this dashboard to:
+* Quickly see number of active apps, clusters, pipelines, and users.
+* See activity levels apps, clusters, pipelines and users
+
+![SpinnakerOverviewDashboard](/assets/images/sumologic-dashboard-overview.png)
+
+### Spinnaker Pipelines Dashboard
+This is a dashboard that contains pipeline execution history and metrics, including most common errors.
+Use this dashboard to:
+* See pipeline execution history and compare the last 14 days to  previous 14 days
```

Comment:
This sounds like the same 14 days. I think you mean "the last 14 days and the 14 days previous" which is awkwardly written but more clear. Let me know if my guess is correct, and I'll work on massaging the sentence. @away168

Reply:
yeah.  it was confusing to write :-D  
so basically comparing the most recent 14 day period, and the 14 day period prior.  so however that turns into proper english works for me.

Reply:
How about
"See pipeline execution history and compare the last 14 days to the 14 days that preceded them."